docs/README.md

```diff
@@ -65,6 +65,22 @@ mtLinkSdk.init(clientId, options);
 | options.mode                               | `production`, `staging`, `develop`, `local` | false    | `production`  | <p>Environment for the SDK to connect to, the SDK will connect to the Moneytree production server by default.<ul><li>Moneytree clients should use `staging` for development as `develop` may contain unstable features.</li><li>`local` should only be used for SDK development as it has local dependencies.</li></ul></p>                                                                                                  |
 | options.locale                             | string                                      | false    | Auto detect.  | Force Moneytree to load content in this specific locale. A default value will be auto detected based on guest langauges configurations and location if available. Check this [spec](https://www.w3.org/TR/html401/struct/dirlang.html#h-8.1.1) for more information.<br /><br />Currently supported values are:<br />`en`, `en-AU`, `ja`.                                                                                    |
 | options.cobrandClientId (private)          | string                                      | false    |               | <strong>NOTE: This is an internal attribute. Please do not use it unless instructed by your integration representative.</strong><br /><br />Brand Moneytree apps with client's branding. E.g: logo or theme.                                                                                                                                                                                                                 |
+| options.samlSubjectId   | string    | false    |   | Set subject Id for saml session version.                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+
+
+### setSamlSubjectId
+Only includes the parameter when calling authorize, logout, openService methods
```

Comment:
Lol, I was expecting more explanation of what the method for instead of just using my note for the other half of the explanation.